tests/features/bootstrap/FeatureContext.php

```diff
@@ -1110,6 +1121,40 @@ public function assertPageNotCached() {
    * @Given I select/check the :row_text row
    */
   public function assertSelectRow($text) {
+    $checkbox = $this->getRowCheckboxByText($text);
+    $checkbox->check();
+  }
+
+  /**
+   * Attempts to uncheck a checkbox in a table row containing a given text.
+   *
+   * @param string $text
+   *   Text in the row.
+   *
+   * @Given I deselect/uncheck the :row_text row
+   */
+  public function assertDeselectRow($text) {
+    $checkbox = $this->getRowCheckboxByText($text);
+    $checkbox->uncheck();
+  }
+
+  /**
+   * Attempts to fetch a checkbox in a table row containing a given text.
+   *
+   * @param string $text
+   *   Text in the row.
+   *
+   * @return \Behat\Mink\Element\NodeElement|null
+   *   The checkbox element.
+   *
+   * @throws \Exception
+   *   If the page contains no rows, no row contains the text or the row
+   *   contains no checkbox.
+   *
+   * @return \Behat\Mink\Element\NodeElement|null
```

Comment:
Since this method should be throwing exceptions when any of the required conditions are not met, it should not return `NULL` under any circumstances.